server/api.go

```diff
@@ -138,8 +138,17 @@ func (a *API) processLifecycle(w http.ResponseWriter, req *http.Request) {
 			if err != nil {
 				a.p.API.LogError("Unable to refresh the subscription", "error", err.Error())
 			}
+		} else if event.LifecycleEvent == "subscriptionRemoved" {
+			_, err := a.p.msteamsAppClient.SubscribeToChannels(a.p.GetURL()+"/", a.p.configuration.WebhookSecret, !a.p.configuration.EvaluationAPI)
+			if err != nil {
+				a.p.API.LogError("Unable to subscribe to channels", "error", err)
+			}
+
+			_, err = a.p.msteamsAppClient.SubscribeToChats(a.p.GetURL()+"/", a.p.configuration.WebhookSecret, !a.p.configuration.EvaluationAPI)
+			if err != nil {
+				a.p.API.LogError("Unable to subscribe to chats", "error", err)
+			}
 		}
-		// TODO: handle "missed" lifecycle event to resync
```

Comment:
we are not handling yet the `missed` event, so I would prefer to keep this comment here to remember that.

Reply:
the resources we are using don't support `missed` lifecycle notification event
you can read more about it here: https://learn.microsoft.com/en-us/graph/webhooks-lifecycle?tabs=http#resources-that-support-lifecycle-notifications